docs/source/support/release-notes.rst

```diff
@@ -12,7 +12,12 @@ HEAD — ongoing
 DAML Integration Kit
 ~~~~~~~~~~~~~~~~~~~~
 
-- Added new CLI flag ``--all-tests`` to the :doc:`Ledger API Test Tool </tools/ledger-api-test-tool/index>` to run all default and optional tests.
+- Added new CLI flag ``--all-tests`` to the :doc:`Ledger API Test Tool
+  </tools/ledger-api-test-tool/index>` to run all default and optional tests.
+- Added new CLI flag ``--command-submission-ttl-scale-factor`` to the
+  :doc:`Ledger API Test Tool </tools/ledger-api-test-tool/index>` to control scale
+  buffer time given for command submission via Max Record Time of submitted
+  transactions.
```

Comment:
Copy docs from command line here. That explanation given there is better.

Reply:
Done